packages/app/src/debug/LayeredBrowserIcons.vue

```diff
@@ -59,6 +60,7 @@ const BROWSER_MAP: Record<BrowserType, any> = {
   'EDGE': IconBrowserEdge,
   'WEBKIT': IconBrowserWebkit,
   'ELECTRON': IconBrowserElectronLight,
+  'CHROME-BETA': IconBrowserChromeBeta,
```

Comment:
This actually needs to be `CHROME BETA` with a space and not a dash to match what is being sent from the Cloud.   Unfortunately, there is no enum value for these at this time.  You can also remove the existing row on line 56 which would be a duplicate.

Reply:
Also, I found the code where the Cloud is mapping browser names to icons.  It is here: https://github.com/cypress-io/cypress-services/blob/46073cb5c387af3b203404280a1664a85e233b93/packages/common/src/logos/getLogoPath.ts

Can you add a comment here with a link to this code for future reference.